curl_cffi/requests/session.py

```diff
@@ -35,11 +35,6 @@
 from .utils import HttpVersionLiteral, not_set, set_curl_options
 from .websockets import AsyncWebSocket, WebSocket
 
-with suppress(ImportError):
-    import gevent
-
-with suppress(ImportError):
-    import eventlet.tpool
```

Comment:
![it's gone now](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExNTNzZDdpdzhyM3Zkb2tnZ3R6YnN1NTJpank0bDlyOXprMTJ1MjhmZiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/omb3kqbCFamkJvCbUp/giphy.gif)